### ISSUE TITLE
move `config.json` to the odoo server

### ISSUE THREAD

**Issue by vaab** [Lokavaluto/monujo#217]
This is a proposition to talk about. My arguments about a `config.json` on the odoo server.

- odoo server address is mandatory for monujo to run, so it would only require this URL to start up.
- all the parameters (mapUrl, cguUrl, helpUrl, css themes) should be changeable by the managers of odoo, easily through a form in odoo's interface. Not in a `compose.yml` which require ssh access, and complex knowledge and high permissions.
- through odoo, the configuration would be updatable for mobile app without making a new mobile release (APK, IPA) each time any param of the `config.json` changes. And all currently installed mobile app and web-app would be updated without any re-installation (I'm talking of update of the config.json content : the skin and the urls).
- currently, logos and other resources are actually already on odoo server.

How it would be done:
- on `compose.yml` side, we would only have to set the odoo URL (and optionally db).
- on odoo's side, through the actual API, we would have a public entry point to request the `config.json`, this would not require a big change in monujo.

For mobile APK:
- the only thing differentiating the apps for each currency, would be the personalization (icon, splash).

**Comment by seegwen** [comment]
+1 I'm all for this proposal.

> the only thing differentiating the apps for each currency, would be the personalization (icon, splash).

And the Odoo entrypoint isn't it ?

**Comment by vaab** [comment]
> And the Odoo entrypoint isn't it ?

indeed, and we could actually offer a generic app (that's not what the currencies asked for, so it would be offered mainly for developpers and advanced users, and would have a generic icon and splash), it would ask for the odoo server URL as first question (with a default theme), it could keep the choice in localstorage for avoiding asking the next time the app is launched... Upon odoo server selection, the app would switch to the currency theme, and stay this way until we switch to another odoo server.

BTW, this is what biletujo does.

And that's a separate concern from this thread.
